moja-kuhinjica-frontend/src/components/restaurant/restaurant_list/RestaurantList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { GridDiv } from '../../../styles/global'
+import RestaurantCard from '../restaurant_card/RestaurantCard'
+
+const RestaurantList = () => {
+    return (
+        <GridDiv>
+            <RestaurantCard />
```

Comment:
You could have put data that will go in `<RestaurantCard />` props in  separate  array, and then just map Restaurant cards

Reply:
Done. I left it like this at first because it is still only a static page.